ui/src/components/executions/Resume.vue

```diff
@@ -17,6 +17,9 @@
             <inputs-form :initial-inputs="inputsList" :execution="execution" v-model="inputs" />
         </el-form>
         <template #footer>
+            <el-button @click="cancel()">
+                {{ $t('cancel') }}
+            </el-button>
```

Comment:
Added a button for cancel and removed submit formating to make it look like other cancel buttons in Kestra.